src/core/agent/services/identifierService.ts

```diff
@@ -375,6 +377,86 @@ class IdentifierService extends AgentService {
     });
     return multisigId;
   }
+
+  async rotateMultisig(ourIdentifier: string): Promise<string | undefined> {
```

Comment:
This should return `Promise<string>` - we should throw in case we cannot complete it, not just not return...